README.md

```diff
@@ -92,12 +95,16 @@ except for `libxmlsec1`, which you can install with Homebrew:
 
 ### Running Tests
 
+Make sure you have [PhantomJS](http://phantomjs.org) installed prior to running tests. Install with the following:
+
+    $ brew install phantomjs
+
 To run all the tests:
 
     $ bundle exec rspec
 
 See RSpec [docs](https://relishapp.com/rspec/rspec-core/docs/command-line) for
-more information.
+more information. 
```

Comment:
Good catch! Thanks for this update. It looks good to me.
One minor nit: could you please get rid of the trailing whitespace you added on line 107? I recommend installing a plugin for your text editor that automatically removes trailing whitespaces when you save a file. For example, here is one for Sublime Text:
https://github.com/SublimeText/TrailingSpaces, and for Atom: https://github.com/atom/whitespace.